api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -153,6 +153,7 @@ private static CustomResourceDefinition crd(Class<? extends CustomResource> cls,
             group = KafkaUser.RESOURCE_GROUP;
             kind = KafkaUser.RESOURCE_KIND;
             listKind = KafkaUser.RESOURCE_LIST_KIND;
+            status = new CustomResourceSubresourceStatusBuilder().build();
```

Comment:
I think you don't need this.

Reply:
`io/strimzi/operator/user/operator/KafkaUserOperatorTest.java` is failing without this.

Reply:
Ah, yes, so it will. But let's change both the occurrence here and the one above for `Kafka` to `new CustomResourceSubresourceStatus()`. There's no point in using a builder to build an empty object. It's simply non-nullness that's needed, I think.